packages/drivers/odsp-driver/src/odspError.ts

```diff
@@ -5,52 +5,6 @@
 
 import { createOdspNetworkError } from "@fluidframework/odsp-doclib-utils";
 import { IOdspSocketError } from "./contracts";
-import { parseAuthErrorClaims } from "./parseAuthErrorClaims";
-
-const nullToUndefined = (a: string | null) => a ?? undefined;
-
-function numberFromHeader(header: string | null): number | undefined {
-    if (header === null) {
-        return undefined;
-    }
-    const n = Number(header);
-    if (Number.isNaN(n)) {
-        return undefined;
-    }
-    return n;
-}
-
-/**
- * Throws network error - an object with a bunch of network related properties
- */
-export function throwOdspNetworkError(
-    errorMessage: string,
-    statusCode: number,
-    response?: Response,
-    responseText?: string,
-): never {
-    const claims = statusCode === 401 && response?.headers ? parseAuthErrorClaims(response.headers) : undefined;
-
-    const networkError = createOdspNetworkError(
-        response && response.statusText !== "" ? `${errorMessage} (${response.statusText})` : errorMessage,
-        statusCode,
-        response ? numberFromHeader(response.headers.get("retry-after")) : undefined, // seconds
-        claims);
-
-    const errorAsAny = networkError as any;
-
-    errorAsAny.response = responseText;
-    if (response) {
-        errorAsAny.type = response.type;
-        if (response.headers) {
-            errorAsAny.sprequestguid = nullToUndefined(response.headers.get("sprequestguid"));
-            errorAsAny.serverEpoch = nullToUndefined(response.headers.get("x-fluid-epoch"));
-        }
-    }
-
-    // eslint-disable-next-line @typescript-eslint/no-throw-literal
-    throw networkError;
-}
 
 /**
  * Returns network error based on error object from ODSP socket (IOdspSocketError)
```

Comment:
maybe move errorObjectFromSocketError (that follows below) also with it?

Reply:
Lets keep it here until some need arises. Actually throwodspNetworkError was duplicated, so we thought of moving it. IOdsoNetworkError which is used here is also defined in contracts. So for now lets keep it here.